examples/shader/Program.cs

```diff
@@ -13,10 +13,7 @@ protected Effect(string name)
             myName = name;
         }
 
-        public string Name
-        {
-            get { return myName; }
-        }
+        public string Name => myName;
```

Comment:
According to the [roadmap doc](https://github.com/dotnet/core/blob/master/roadmap.md), Q3, this year.